web/server/codechecker_server/api/report_server.py

```diff
@@ -2004,6 +2032,22 @@ def getRunResults(self, run_ids, limit, offset, sort_types,
                         report.review_status_date,
                         report.review_status_is_in_source)
 
+                    blame_info = blame_infos[report.id] \
+                        if report.id in blame_infos else None
+                    if blame_info and blame_info.commits and blame_info.blame:
+                        blame_data = [b for b in blame_info.blame
+                                      if report.line >= b.startLine
+                                      and report.line <= b.endLine]
+                        commitHash = blame_data[0].commitHash \
+                            if len(blame_data) else None
+                        commitInfo = {cHash: commit for cHash, commit
+                                      in blame_info.commits.items()
+                                      if cHash == commitHash}
+                        blame_info = BlameInfo(
+                            commits=commitInfo,
+                            blame=blame_data
+                        )
+
```

Comment:
These code snippets are basically copy-paste from above, right? Can we do something about that?

Reply:
It is more readable form.